### ISSUE TITLE
[Main UI] Dark Mode not applied on sign in screen.

### ISSUE THREAD

**Issue by Skinah** [openhab/openhab-webui#385]
Steps to reproduce under Build #1964. EDIT: Only used Edge browser to test.

1. Set 'dark mode' by clicking on 'help and about'. Really don't like that 'Appearance Options' are hidden under this menu.
2. Sign out of the session by clicking on user name at bottom left corner of menu bar.
3. Click on the lock marked as "Sign in as an administrator to access settings" down bottom left corner.
4. Screen is now bright white and not in dark mode.

I expected the color to stay dark and not go white.

**Comment by wborn** [comment]
That part currently only uses the `prefers-color-scheme` value of your browser, see:
* https://github.com/openhab/openhab-webui/issues/309
* https://github.com/openhab/openhab-core/pull/1600

I know from first hand experience that for instance Chrome on Ubuntu does not properly detect your dark mode preference and you have to start it with a few extra options to use it:

```bash
/usr/bin/google-chrome-stable --force-dark-mode --enable-features=WebUIDarkMode
```

See also: https://bugs.chromium.org/p/chromium/issues/detail?id=998903

**Comment by florian-h05** [comment]
Closing because this is no MainUI issue, but a browser issue. I had this problem with my Chromium on Fedora 36/KDE Plasme as well, but it is now fixed (current version is `110.0.5481.100 (Offizieller Build) Fedora Project (64-Bit)`).
